frontend/src/api/index.js

```diff
@@ -1,10 +1,13 @@
 const METABACKEND_URL = 'http://0.api.getline.in';
 const METABACKEND_NETWORK = '4';
 let api;
```

Comment:
Why not make this a static member of the class?

Reply:
I prefer to use JS modules. If I had a class, I would have to use new Keyword, which I'd like to avoid